hasura/metadata/databases/default/tables/public_users.yaml

```diff
@@ -188,6 +188,22 @@ event_triggers:
       paused: true
       schedule: 0 0 * * *
       timeout: 60
+  - name: insertUserOrgMember
+    definition:
+      enable_manual: false
+      insert:
+        columns: '*'
+      update:
+        columns:
+          - deleted_at
+    retry_conf:
+      interval_sec: 10
+      num_retries: 5
+      timeout_sec: 60
+    webhook: '{{HASURA_API_BASE_URL}}/event_triggers/eventManager?event=insertUserOrgMember'
```

Comment:
@sshmm It looks like you are doing two different checks via the same event_trigger:
`insertOrgMember` on add user to circle and 
`deleteOrgMember` on delete user from circle.

I think it would be cleaner to have two smaller, simpler, separate event_triggers to handle each of these cases separately.

Reply:
after removing the deleting part, insertUserOrgMember became much simpler